package.json

```diff
@@ -145,7 +146,9 @@
     "tslint-no-circular-imports": "^0.7.0",
     "tslint-sonarts": "1.9.0",
     "typescript": "3.8.3",
-    "verdaccio": "4.5.1"
+    "verdaccio": "4.5.1",
+    "webpack": "4.42.0",
+    "webpack-sources": "1.4.3"
```

Comment:
At its core, this is needed because `rules_nodejs` doesn't understand yarn workspaces. It's not super clear to me whether it should though, or how yarn pnp plays into that either.

We could reduce this burden with some health checks on our repository, like checking that these versions are the same everywhere they are mentioned, plus some exceptions. Maybe at that point what we need to to also loop in dev-infra to share that tooling between fw/comp/cli. We're the only ones that use yarn workspaces afaik but we're not the only ones that have versions sprinkled around that need to be matched.

Reply:
For now, I think extra validation checks are going to be paramount.  Especially to ensure that the tests are using the same dependency versions that the users are using when they install the package.

I think this is also problematic if we need to use a different version of a dependency for a particular package (recent build optimizer TS issue, for example) or is there a mechanism for that situation?  I don't think there are any of these currently though.

Reply:
cc @josephperrott

Reply:
We should also update renovate configuration now, so that it doesn't create separate PRs for dependencies and dev deps, as otherwise we'll have 2 PRs for each dependency update.